benches/src/bin/collect.rs

```diff
@@ -522,41 +553,45 @@ impl State {
                 let mut samples = samples
                     .iter()
                     .filter_map(|sample| {
-                        Some((ids.remove(sample)?, throughput.get(sample).copied()?))
+                        let mean = ids.remove(sample)?;
+                        let throughput = *throughput.get(sample)?;
+                        Some((mean, throughput))
+                    })
+                    .map(|(mean, throughput)| {
+                        let ratio = map_to_ratio(baseline, mean);
+                        (throughput, ratio)
                     })
-                    .map(|(mean, t)| (t, map_to_ratio(baseline, mean)))
                     .collect::<Vec<_>>();
-                samples.sort_unstable_by_key(|(_, mean)| *mean);
+                samples.sort_unstable_by_key(|(_, ratio)| *ratio);
                 (name.clone(), samples)
             })
             .collect::<Vec<_>>();
 
         if all {
-            let iter = dependent_groups.into_iter().map(|(name, x_y)| {
-                groups.remove(&name);
-                let samples = x_y
-                    .iter()
-                    .map(|(x, y)| Sample {
-                        throughput: *x,
-                        time: *y,
-                    })
-                    .collect();
-                let (_, dep_per_unit) = dependent_cost(&name, x_y);
-                (
-                    name,
-                    Cost::DependentAll {
-                        samples,
-                        dep_per_unit,
-                    },
-                )
-            });
-            costs.0.extend(iter);
+            // let iter = dependent_groups.into_iter().map(|(name, x_y)| {
+            //     groups.remove(&name);
+            //     let samples = x_y
+            //         .iter()
+            //         .map(|(x, y)| Sample {
+            //             throughput: *x,
+            //             time: *y,
+            //         })
+            //         .collect();
+            //     let (_, dep_per_unit) = dependent_cost(&name, x_y);
+            //     (
+            //         name,
+            //         Cost::DependentAll {
+            //             samples,
+            //             dep_per_unit,
+            //         },
+            //     )
+            // });
+            // costs.0.extend(iter);
```

Comment:
I guess we need it back?=D

Reply:
I removed this section because it is unclear to me how this will be used going forward, or if we still use it in our current benchmark process. This code is designed not to be used when generating formatted output (e.g., `-f rust`) so we don't use it, and it is just unmaintained code after this point.